config/samples/crowdstrike_v1alpha1_falconsensor.yaml

```diff
@@ -4,6 +4,23 @@ metadata:
   name: falcon-helm
 spec:
   # Default values copied from <project_dir>/helm-charts/falcon-sensor/values.yaml
+  container:
+    certExpiration: 3650
```

Comment:
Should this be a variable?

Reply:
For our NIST friends, for example

Reply:
This is the variable. It's just the default value for the variable. ;)